### DISCUSSION TITLE
Schema extendability

### DISCUSSION THREAD

**Discussion by mitsos1os** [prisma/prisma#7456] [Q&A] [answered]
Hi everyone.
Is there a way to define a set of properties in the schema file which are shared in different models?

For example let's say I have the set of properties `createdAt`, `updatedAt`, `deletedAt` each one serving its obvious purpose.
Is there a way to set these properties on different models, except for manually defining every property in each model?

This would be quite useful

**Comment by janpio** [comment, marked as answer]
Not yet, a similar feature request is tracked here: https://github.com/prisma/prisma/issues/2371

**Reply by mitsos1os** [reply]
Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
